config/activator.yaml

```diff
@@ -112,3 +112,22 @@ spec:
         - name: config-observability
           configMap:
             name: config-observability
+---
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+    name: activator
+    namespace: knative-serving
+spec:
+    minReplicas: 1
+    maxReplicas: 20
+    scaleTargetRef:
+      apiVersion: apps/v1
+      kind: Deployment
+      name: activator
+    metrics:
+    - type: Resource
+      resource:
+        name: cpu
+        # Percentage of the requested CPU
```

Comment:
Just to confirm, this means that when we exceed `300m` of CPU we will scale up?

Reply:
Yes